Content/DerivePublicKeys/derivePublicKeys.json

```diff
@@ -1,11 +0,0 @@
-{
-	"derivePublicKeys_titleCreateAccount": "Creating Account",
-	"derivePublicKeys_titleCreatePersona": "Creating Persona",
-	"derivePublicKeys_titleImportLegacyAccount": "Deriving Accounts",
```

Comment:
you can also remove 
`accountRecoveryScan_derivingAccounts`

Reply:
done